packages/react/src/ErrorBoundary.spec.tsx

```diff
@@ -167,6 +167,98 @@ describe('<ErrorBoundary/>', () => {
     expect(screen.queryByText(ERROR_MESSAGE)).not.toBeInTheDocument()
     expect(onReset).toHaveBeenCalledTimes(1)
   })
+
+  it('should catch Error by many criteria', async () => {
```

Comment:
I wish “many criteria” were more specific. There are times when I read the interface while looking at the test code, so I think I can use it better if the test cases are concrete!

Reply:
@sonsurim If you okay, Could you make PR to make our code more clear than before? I want to see what you want to do exactly as code level